tests/modules/nf-core/bases2fastq/test.yml

```diff
@@ -5,14 +5,14 @@
   files:
     - path: output/bases2fastq/output/Bases2Fastq-Sim_QC.html
     - path: output/bases2fastq/output/Metrics.csv
-      md5sum: 0ec6da2b82e191098283474356024abf
+      md5sum: a8094b308b5653071ac029a27733e4a6
```

Comment:
why do the checksums change if no version change was done?

Reply:
Fair question. I got the impression from @Emiller88 that it has something to do with this ongoing [PR](https://github.com/nf-core/modules/pull/3558). I asked Edmund whether I should update the md5sums, and he agreed, and so I did, and the tests worked :-)

Reply:
I still don't understand how this got into master if the md5sums were failing- unless someone merged a failing PR somewhere.

Reply:
As-is it at least seems possible with our current mechanisms that changes to test data could do this though. We should [close that loophole](https://nfcore.slack.com/archives/C043UU89KKQ/p1690903004273109) even if it's not actually the culprit here.